src/app.js

```diff
@@ -62,10 +63,13 @@ class App {
         this.timer = new Timer();
 
         this.notifier = new Notifier(this);
-        // at this stage we only work with slack
+        // at this stage we only work with slack or telegram
         if (this.config.SLACK_WEBHOOK_URL) {
             this._slackNotifier = new SlackNotifier(this, {timeout: 3000});
             this.notificationJobs = new NotifierJobs(this);
+        } else if (this.config.TELEGRAM_BOT_TOKEN && this.config.TELEGRAM_CHAT_ID) {
+            this._telegramNotifier = new TelegramNotfier(this);
+            this.notificationJobs = new NotifierJobs(this);
```

Comment:
I don't see a reason to not support both in the same instance. Is up to operator to pick, slack, telegram or both.

The `this.notificationJobs = new NotifierJobs(this);` should be only created once in the case of both being running.

Reply:
True, I somehow thought that it would require changes in the NotifierJobs.